src/lib/orionld/mongoCppLegacy/mongoCppLegacySubscriptionDelete.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SRC_LIB_ORIONLD_MONGOCPPLEGACY_MONGOCPPLEGACYSUBSCRIPTIONDELETE_H_
+#define SRC_LIB_ORIONLD_MONGOCPPLEGACY_MONGOCPPLEGACYSUBSCRIPTIONDELETE_H_
+
+/*
+*
+* Copyright 2019 FIWARE Foundation e.V.
+*
+* This file is part of Orion-LD Context Broker.
+*
+* Orion-LD Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion-LD Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion-LD Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* orionld at fiware dot org
+*
+* Author: Gabriel Quaresma
+*/
+
+extern "C"
+{
+#include "kjson/KjNode.h"                                        // KjNode
+}
```

Comment:
You don't need to include `KjNode.h` - `KjNode` isn't used in this file - remove the inclusion